rules/aip0158/response_repeated_first_field_test.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aip0158
+
+import (
+	"testing"
+
+	"github.com/googleapis/api-linter/rules/internal/testutils"
+)
+
+func TestResponseRepeatedFirstField(t *testing.T) {
```

Comment:
Can you add a test for empty message? Like:
```protobuf
message ListBooksResponse{}
```

Reply:
What's the right situation here? I'd argue that a ListBooksResponse should **require** that the first field be repeated. If it's missing, then that seems like a failure to me. Currently we will ignore this and say "OK, no `next_page_token` field, so therefore this is perfectly fine."

Reply:
I added warning message for an empty response